helm/happa-chart/templates/ingress.yaml

```diff
@@ -13,6 +13,8 @@ metadata:
     {{- if and (eq .Values.Installation.V1.Provider.Kind "aws") (.Values.Installation.V1.Security.RestrictAccess.GSAPI) }}
     nginx.ingress.kubernetes.io/whitelist-source-range: "{{ .Values.Installation.V1.Security.Subnet.VPN }},{{ .Values.Installation.V1.Security.Subnet.Customer }}"
     {{- end }}
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      add_header X-Frame-Options deny always;
```

Comment:
This is the thing that really matters.

The rest is a bit of cleanup to hopefully make things less confusing next time around.